src/main/java/com/iota/iri/utils/dag/TraversalConsumer.java

```diff
@@ -0,0 +1,12 @@
+package com.iota.iri.utils.dag;
+
+import com.iota.iri.controllers.TransactionViewModel;
+
+/**
+ * Functional interface for the lambda function getting passed into the DAGUtils traverse method to process the found
+ * transactions.
+ */
+@FunctionalInterface
+public interface TraversalConsumer {
```

Comment:
There is also a `Consumer` built in.
Once again I am totally fine with the code.